README.md

```diff
@@ -3,7 +3,7 @@ This is n this project l set up a JavaScript project for the Leaderboard list ap
 
 In this project, you will be using javascript arrays and object to store the list of books and saving the book colection in the local storage. Do not use any other framework, just pure javascript. After you are done with the project, run tests and deploy.
 
-### [Live Demo Link](https://bushmusi.github.io/todo-list/)
+### [Live Demo Link](https://bushmusi.github.io/leaderboard/)
```

Comment:
- [ ] Your app works as expected, however, your README needs a little improvement so that you can have a more professional README. Kindly fix the typo you have in the project description and make the sentence more readable. This is what you have there 👉  `This is n this project l set up a JavaScript project for the Leaderboard list app, using webpack and ES6 features, notably modules..` 

- [ ] The screenshot path is broken, kindly fix it so that it displays correctly

<img width="128" alt="chrome_yta8iDIdwf" src="https://user-images.githubusercontent.com/54848363/158018098-bd0d4459-fba7-4998-af06-14531eb759be.png">

Reply:
Hi @DammyShittu 

Noted and fixed, Thanks 🤔